pages/api/revalidate.js

```diff
@@ -33,17 +30,16 @@ async function handler(req, res) {
     return res.status(500).json({ error: `The parameter "uri" is required.` });
   }
 
-  try {
-    if (APP_ROUTER_REVALIDATE.includes(query.uri)) {
-      const searchParams = new URLSearchParams(query);
-      await fetch(`${BASE_URL}/api/app-revalidate?${searchParams.toString()}`);
-      return res.status(200).json({ revalidated: true });
-    } else {
-      await res.revalidate(`/${query.uri}`);
-      return res.status(200).json({ revalidated: true });
-    }
-  } catch (error) {
-    console.error(error);
+  const searchParams = new URLSearchParams(query);
+  const response = await fetch(
```

Comment:
forward all revalidates to the app directory route handlers. Next step will be to delete this endpoint and rename the app directory one to take over